omp/distributed/partition_kernels.cpp

```diff
@@ -60,6 +60,8 @@ void build_starting_indices(std::shared_ptr<const DefaultExecutor> exec,
     auto size_per_thread =
         static_cast<size_type>(ceildiv(num_ranges, num_threads));
     vector<LocalIndexType> local_sizes(num_parts * num_threads, 0, {exec});
+    // TODO: PGI/NVHPC(22.7) doesn't like reduction with references
+    int tmp_reduction = 0;
```

Comment:
should we do it in this pr? We do not support omp with PGI/NVHPC, so we can just mention somewhere without changing it

Reply:
fair enough I can remove this